pkg/controller/velero/velero.go

```diff
@@ -39,21 +39,10 @@ const (
 	veleroGcpImageTag = "velero-plugin-for-gcp:v1.0.1"
 )
 
-func (r *ReconcileVelero) provisionVelero(reqLogger logr.Logger, namespace string, platformStatus *configv1.PlatformStatus, instance *veleroInstallCR.VeleroInstall) (reconcile.Result, error) {
+func (r *ReconcileVelero) provisionVelero(reqLogger logr.Logger, namespace string, instance *veleroInstallCR.VeleroInstall) (reconcile.Result, error) {
 	var err error
 
-	var locationConfig map[string]string
-	switch r.driver.GetPlatformType() {
-	case configv1.AWSPlatformType:
-		locationConfig = map[string]string{
-			"region": platformStatus.AWS.Region,
-		}
-	case configv1.GCPPlatformType:
-		// No region configuration needed for GCP
-	default:
-		return reconcile.Result{}, fmt.Errorf("unable to determine platform")
-	}
-
+	locationConfig := r.driver.GetLocationConfig()
```

Comment:
This is sneaky and will actually break things.

So locationConfig starts as an uninitialized map here. It needs to actually be nil on GCP for this to work, not an empty map.

Reply:
Ok, pushed a new commit to try to correct that.